exercises/concept/assembly-line/assembly_line_test.rb

```diff
@@ -3,43 +3,43 @@
 
 class AssemblyLineTest < Minitest::Test
   def test_production_rate_per_hour_for_speed_one
-    assert_equal 221.0, AssemblyLine.new(1).production_rate_per_hour
+    assert_in_delta 221.0, AssemblyLine.new(1).production_rate_per_hour, 0.000_000_0001
```

Comment:
I like the delta check.  Should it be to 3 decimal places, or perhaps 4, since we seems to be working with percentages?  Low precision problem, high value delta.

Reply:
Thanks for the feed back. I updated both, the asserts and the PR description.